src/forum-tool-selector-settings/ForumToolSelectorSettings.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import StudioApiService from '../data/services/StudioApiService';
+
+var selectedForumTool; // eslint-disable-line no-unused-vars
+var setSelectedFunctions = [];
+
+function resetSelected() {
+  for (var i = 0; i < setSelectedFunctions.length; i++) {
+    setSelectedFunctions[i](false);
+  }
+}
+
+function ForumCard(props) {
```

Comment:
Destructure these props as:
```suggestion
function ForumCard({locked, logo, text, support}) {
```
Better names would be, available, logoUrl, description, supportLevel